tla/consensus/Traceccfraft.tla

```diff
@@ -92,11 +87,8 @@ TraceAppendEntriesBatchsize(i, j) ==
 TraceInitReconfigurationVars ==
     /\ reconfigurationCount = 0
     /\ removedFromConfiguration = {}
-    \* Weaken  ccfraft!InitReconfigurationVars  to allow a node's configuration to be initially empty.
-     \* This seems to be a quirk of raft_driver (related RaftDriverQuirk).
-    /\ configurations = [ s \in Servers |-> IF s = TraceLog[1].msg.state.node_id 
-                                            THEN ToConfigurations(<<TraceLog[1].msg.new_configuration>>)
-                                            ELSE [ j \in {0} |-> {} ] ]
+    /\ LET startNode == TraceLog[1].msg.state.node_id
+       IN CCF!StartState(startNode, TraceServers)
```

Comment:
Given that `Traceccfraft` extends `ccfraft`, just `StartState` should also work.

Reply:
I think I tried that, but I'll try again.